### ISSUE TITLE
asyncio should not put under requirements.txt

### ISSUE THREAD

**Issue by RyanSiu1995** [okta/okta-sdk-python#157]
Hi community,
I am now using python 3.7+ with `pip-compile`.
And I will get error with asyncio if I installed it from pip.
From the doc in asyncio, 
```
asyncio requires Python 3.3 or later! The asyncio module is part of the Python standard library since Python 3.4.
```
And also,
```
This version is only relevant for Python 3.3, which does not include asyncio in its stdlib.
```
So, my suggestion here is to remove the asyncio and release the okta package with 3.3+ only.
Thank you!

**Comment by serhiibuniak-okta** [comment]
Hi @RyanSiu1995 , thank you for this issue and PR, currently we're on discussion on this here - https://github.com/okta/okta-sdk-python/pull/149. I believe, we will cover it with the first PR, will keep you posted.

**Comment by RyanSiu1995** [comment]
Hi @sergiishamrai-okta, I don't see the update in the #149. Maybe you can first check out #158 and merge it to master.
It is a simple removal in the `asyncio`. Then, #149 can be rebased from the change in #158.

**Comment by serhiibuniak-okta** [comment]
Hi @RyanSiu1995 , I've included both commits into one and just merged to master - https://github.com/okta/okta-sdk-python/pull/159. Need to wait until we can release v1.2.2. Thanks for your contribution.

**Comment by serhiibuniak-okta** [comment]
@RyanSiu1995 we decided to include some additional changes, so we released v1.3.0. It is ready. I'm closing this issue as resolved. Thanks again.
